modules/ding_campaign_plus/ding_campaign_plus.features.field_instance.inc

```diff
@@ -45,6 +45,12 @@ function ding_campaign_plus_field_default_field_instances() {
         'type' => 'hidden',
         'weight' => 0,
       ),
+      'teaser_no_overlay' => array(
+        'label' => 'above',
+        'settings' => array(),
+        'type' => 'hidden',
+        'weight' => 0,
+      ),
```

Comment:
These seem unrelated, but this is how features exports it so better keep it as is.